### DISCUSSION TITLE
CSS not showing on webserver

### DISCUSSION THREAD

**Discussion by V0lantis** [apache/airflow#25849] [Q&A] [answered]
Hello,

I have been trying to work on the webserver on some issue with observed at my company. When I tried to launch the dev environment with `breeze start-airflow`, the webserver didn't show normally with its CSS working, as it usually does. 

<img width="1424" alt="Capture d’écran 2022-08-20 à 22 12 16" src="https://user-images.githubusercontent.com/37664438/185764679-ee30f9cb-52a8-452c-a8c6-ba590e2b74a6.png">
I have been trying multiple different commands without any success, like:

* `yarn run dev` or `yarn run prod` inside`airflow/www`
* `breeze compile-www-assets`, but without any improvement. 
* I even tried to go in `airflow/ui` and start the webserver, but it didn't work either unfortunately.

What I am doing wrongly? I didn't read through all documentations though I tried to look for some key words. I somehow didn't find anyone else which seems to encounter the same issue in the QA or in the issues.

thanks for your help,
cheers.

**Comment by potiuk** [comment, marked as answer]
Are you using latest main? If so, it works fine for me. What code version are you using (Breeze is intended to work only for latest main version because it is a development environment for airflow, so it you try to run it with old version source code, it might or might not work.

**Reply by V0lantis** [reply]
Thank you @potiuk as always. It was indeed the problem. I thought I pulled the latest changes, but it seemed not. So I erased everything and started from 0
